### ISSUE TITLE
train-my dataset

### ISSUE THREAD

**Issue by e871** [lhotse-speech/lhotse#933]
hello, i have a dataset with audio and labels. can i prepare my own dataset with this package?

**Comment by pzelasko** [comment]
Yes, please check out the tips here https://lhotse.readthedocs.io/en/latest/corpus.html#adding-new-corpora and feel free to use any of the existing recipes as a starting point https://github.com/lhotse-speech/lhotse/tree/master/lhotse/recipes
